cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Committee.hs

```diff
@@ -139,16 +135,19 @@ pGovernanceCommitteeCreateColdKeyResignationCertificateCmd era = do
   mkParser w =
     GovernanceCommitteeCreateColdKeyResignationCertificateCmd <$>
       (GovernanceCommitteeCreateColdKeyResignationCertificateCmdArgs w <$>
-        coldVKeyOrFileOrScriptHash <*> pAnchor <*> pOutputFile)
-  coldVKeyOrFileOrScriptHash =
-    asum
-      [ VkhfshKeyHashFile . VerificationKeyOrFile <$> pCommitteeColdVerificationKeyOrFile
-      , VkhfshKeyHashFile . VerificationKeyHash <$> pCommitteeColdVerificationKeyHash
-      , VkhfshScriptHash <$>
-          pScriptHash
-            "cold-script-hash"
-            "Cold Native or Plutus script file hash (hex-encoded). Obtain it with \"cardano-cli conway governance hash script ...\"."
-      ]
+        pColdCredential <*> pAnchor <*> pOutputFile)
+
+pColdCredential :: Parser (VerificationKeyOrHashOrFileOrScriptOrScriptHash CommitteeColdKey)
+pColdCredential =
+  asum
+    [ VkhfsshKeyHashFile . VerificationKeyOrFile <$> pCommitteeColdVerificationKeyOrFile
+    , VkhfsshKeyHashFile . VerificationKeyHash <$> pCommitteeColdVerificationKeyHash
+    , VkhfsshScriptHash <$>
+        pScriptHash
+          "cold-script-hash"
+          "Cold Native or Plutus script file hash (hex-encoded). Obtain it with \"cardano-cli conway governance hash script ...\"."
+    , VkhfsshScript <$> pScriptFor "cold-script-file" Nothing "Cold Native or Plutus script file"
```

Comment:
I think it would be useful to but "committee" in the help text.

Reply:
Do you have any suggestion how to phrase it?

Reply:
"Committee cold Native or Plutus script file"